kalite/distributed/templates/distributed/base.html

```diff
@@ -82,9 +84,26 @@
 
         <script type="text/javascript" src="{% static 'js/fastclick.js' %}"></script>
         <script>
+
         $(function() {
             FastClick.attach(document.body);
         });
+
+         /* TODO(jtamiace): Backbonify this function and this page... when the time comes.
```

Comment:
Yep, got until the end of the quarter!

On Tue, 7 Apr 2015 at 13:50 jtamiace <notifications@github.com> wrote:

> In kalite/distributed/templates/distributed/base.html
> <https://github.com/learningequality/ka-lite/pull/3476#discussion_r27918904>
> :
>
> >          $(function() {
> >              FastClick.attach(document.body);
> >          });
> > +
> > +         /* TODO(jtamiace): Backbonify this function and this page... when the time comes.
>
> @rtibbles <https://github.com/rtibbles> Definitely will tackle this when
> I'm not working on content curation; great chance to practice backbone.
> There's a couple of months left in this cycle right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3476/files#r27918904>.
>